src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/details.html.twig

```diff
@@ -36,7 +36,7 @@
 {% endset %}
 
 {% set merchantReturnsTitle %}
-  {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }} ({{ orderForViewing.returns.orderReturns|length }})
+  {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }} (<span class="count">{{ orderForViewing.returns.orderReturns|length }}</span>)
```

Comment:
```suggestion
  {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }} (<span class="js-test-count">{{ orderForViewing.returns.orderReturns|length }}</span>)
```

If only used by QA, maybe we could add this.

Reply:
Done thx :+1: